kannon/master.py

```diff
@@ -1,9 +1,10 @@
+from __future__ import annotations
+
 import logging
 import os
 from collections import deque
 from copy import deepcopy
 from time import sleep
-from typing import Deque, Dict, List, Optional, Set
```

Comment:
[refactoring]

these types are deprecated on 3.9

https://docs.python.org/3.9/library/typing.html#typing.Dict

add `from __future__ import annotations` to support 3.8